platform/ios/PolyPodApp/FeatureContainer/initIframe.js

```diff
@@ -1,13 +1,11 @@
-const {port1, port2} = new MessageChannel();
+const { port1 } = new MessageChannel();
 
 function initIframe(el) {
-  port1.start();
-  port1.onmessage = event => {
-    // incoming msgpack-encoded events:
-    webkit.messageHandlers.event.postMessage(event.data);
-  };
-  el.contentWindow.postMessage("", "*", [port2]);
-
-  // outgoing msgpack-encoded events:
```

Comment:
Additionally, the fact that there was some dead code here is a hint that the code above might be also non-functional.